plugins/woocommerce/legacy/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "woocommerce-legacy-assets",
+	"version": "1.0.0",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"private": true,
+	"devDependencies": {
```

Comment:
Now that we've moved Grunt into this package, can we remove some of these `devDependencies` from `plugins/woocommerce/package.json`?